### PR TITLE
Geekpete metricbeat docs docker healthcheck 1

### PR DESCRIPTION
**Pull request by geekpete**
Docker healthcheck metrics might prove confusing if it's not understood that the healthcheck metrics 
are only available for containers using images that were created with the docker `HEALTHCHECK`
instruction:
https://docs.docker.com/engine/reference/builder/#healthcheck

These are not generic docker container healthchecks that metricbeat can monitor for all containers.
A container with a healthcheck specified, has a health status in addition to its normal status.

### CODE REVIEW

**Comment by ruflin** [comment]
@geekpete Thanks for the PR. Could you run `make update` in the Metricbeat directory?

File: metricbeat/module/docker/healthcheck/_meta/fields.yml [hunk 1]
```diff
@@ -2,6 +2,8 @@
   type: group
   description: >
     Docker container metrics.
```
Comment (ruflin) [inline metricbeat/module/docker/healthcheck/_meta/fields.yml]: This seems to be off. Not introduced in this PR but I wonder if we should change it to `Docker healthcheck metrics.` 

Should we add an empty line below or do you want to have it all in one paragraph?

**Comment by geekpete** [comment]
Hitting an issue with:

```
make update
Generating NOTICE
Get the licenses available from ['./vendor', './metricbeat/vendor', './metricbeat/module/mysql/vendor', './metricbeat/module/postgresql/vendor', './metricbeat/module/vsphere/vendor', './metricbeat/module/kvm/vendor']
Traceback (most recent call last):
  File "dev-tools/generate_notice.py", line 394, in <module>
    dependencies = create_notice(notice, args.beat, args.copyright, vendor_dirs, args.csvfile, overrides=overrides)
  File "dev-tools/generate_notice.py", line 211, in create_notice
    dependencies = gather_dependencies(vendor_dirs, overrides=overrides)
  File "dev-tools/generate_notice.py", line 69, in gather_dependencies
    lib["license_summary"] = detect_license_summary(lib["license_contents"])
  File "dev-tools/generate_notice.py", line 311, in detect_license_summary
    content = content.replace(b'\xe2\x80\x9c', '"').replace(b'\xe2\x80\x9d', '"')
TypeError: replace() argument 1 must be str, not bytes
make: *** [Makefile:143: notice] Error 1
```

What might I be missing here?
Have ensured I have a recent pull of master branch and tested with both:
- Python 2.7.15
- Python 3.6.7

**Comment by ruflin** [comment]
I think that is a python version problem. It should be 2.7. My assumption is that even when you use 2.7, it will somehow use parts of 3.6. I remember seeing this with others in the past but don't remember the best fix (except wiping 3.6 off which is probably not an option).

**Comment by geekpete** [comment]
I'll investigate further thanks.
I'd tried using 2.7 via a virtualenv.

File: metricbeat/module/docker/healthcheck/_meta/fields.yml (hunk 1 above)
Comment (geekpete) [reply metricbeat/module/docker/healthcheck/_meta/fields.yml]: Docker healthcheck metrics sounds better and more specific to describe it.
Will adjust.

**Comment by geekpete** [comment]
Still just wrestling with my local environment for this one, once I have it working or use an alternative system I'll push the `make update`changes to this branch.

**Comment by ruflin** [comment]
@geekpete If it helps, I could also run `make update` on my end and potentially push to your branch?

**Comment by geekpete** [comment]
That might be good until I sort out my local environment, thanks.

**Comment by ruflin** [comment]
@geekpete If you add me as collaborator, I can push to your repo / branch.

**Comment by ruflin** [comment]
@geekpete Finally pushed to your branch and rebased on master. Lets see what happens.

**Comment by ruflin** [comment]
@geekpete Thanks for the change. Sorry too me a while to get it in. Here is the backport.

**Comment by geekpete** [comment]
Thanks! I'll sort my local environment out so I can actually do the update next time.

**Pull request merged**